### PR TITLE
Closes AgileVentures/MetPlus_tracker#223

### PR DESCRIPTION
**Pull request by patmbolger**
List jobs for job seeker, paginated view in:

1. JS home page
2. JS show page

### CODE REVIEW

File: app/controllers/job_seekers_controller.rb [hunk 1]
```diff
@@ -89,6 +90,7 @@ def home
     @newjobs = Job.new_jobs(@jobseeker.last_sign_in_at).paginate(:page => params[:page], :per_page => 5)
     @past_week = Job.new_jobs(Time.now - 3.weeks).paginate(:page => params[:page], :per_page => 5)
 
+    @application_type = params[:application_type] || 'my-applied'
```
Comment (joaopapereira) [inline app/controllers/job_seekers_controller.rb]: Are we planning on having this sent by parameters? If not we could just set the values directly in the view

File: app/controllers/job_seekers_controller.rb [hunk 2]
```diff
@@ -97,8 +99,22 @@ def index
 
   def show
     @jobseeker = JobSeeker.find(params[:id])
+    @application_type = params[:application_type] || 'js-applied'
```
Comment (joaopapereira) [inline app/controllers/job_seekers_controller.rb]: Are we planning on having this sent by parameters? If not we could just set the values directly in the view

File: app/controllers/job_seekers_controller.rb [hunk 3]
```diff
@@ -97,8 +99,22 @@ def index
...
   end
 
+  def applied_jobs
```
Comment (joaopapereira) [inline app/controllers/job_seekers_controller.rb]: Not 100% sure if this should be here or should be in the Jobs controller, but as you added the view in the job maybe we should move the function there

**Comment by joaopapereira** [comment]
Please review my comments

File: app/controllers/job_seekers_controller.rb (hunk 1 above)
Comment (patmbolger) [inline app/controllers/job_seekers_controller.rb]: Yes, the pagination SW will send this via parameter to the controller.

File: app/controllers/job_seekers_controller.rb (hunk 2 above)
Comment (patmbolger) [inline app/controllers/job_seekers_controller.rb]: See comment above.

File: app/controllers/job_seekers_controller.rb (hunk 3 above)
Comment (patmbolger) [inline app/controllers/job_seekers_controller.rb]: Lets discuss.  This is the action to show applied jobs for a specific job seeker.  The JS ID is passed in via params.  I put the view under jobs as most of the actual fields we display are Job attributes - so it felt more natural to put the view there.

**Comment by joaopapereira** [comment]
After discussion in the Standup we decided the PR implementation was good to go

**Pull request merged**